index.test.js

```diff
@@ -0,0 +1,70 @@
+const { withSVG, withRasterImages, withPlayback } = require('./index.js');
+
+const options = {
+    dev: true,
+    isServer: true,
+};
+
+let config = {
+    module: {
+        rules: [],
+    },
+};
+
+const mockFunction = jest.fn();
+
+afterEach(() => {
+    config = {
+        module: {
+            rules: [],
+        },
+    };
+
+    jest.clearAllMocks();
+});
+
+it('should return a single rule', () => {
+    const arrayOfSVGRules = [];
+    const arrayOfImagesRules = [];
+    const arrayOfPlaybackRules = [];
+
+    arrayOfSVGRules.push(withSVG()().webpack(config, options).module.rules);
```

Comment:
Just an idea, maybe it would be better to snapshot the result. You can check that the snapshot was well created and has all necessary data. This way, if even one little thing changes, the test fails and we need to check if that little change was intended and might not create a big problem.